packages/nodes-base/nodes/Gitlab/GitlabTrigger.node.ts

```diff
@@ -195,7 +195,7 @@ export class GitlabTrigger implements INodeType {
 				try {
 					await gitlabApiRequest.call(this, 'GET', endpoint, {});
 				} catch (error) {
-					if (error.cause.httpCode === '404') {
+					if (error.description.includes('404')) {
```

Comment:
should we add this as OR
`if (error.cause.httpCode === '404' || error.description.includes('404')) {`

Reply:
Changed it accordingly